admin_manual/appliance/Collabora.rst

```diff
@@ -0,0 +1,23 @@
+=======================================
+How to use Collabora with the appliance
+=======================================
+
+Connect to your appliance via IP or FQDN
+
+for example 172.16.40.100 or my_owncloud_appliance
+
+Login in to the management console
+
+for example click on the domain and system settings
+
+type in the Administrator as username and the Password you set
+
+Go to Applicantions/Software and then Appcenter
+
+Search for Collabora Online Development Editon or select it from the applist
+
+Install the C.O.D.E
+
+Now you can use Collabora within ownCloud.
+
+Start by creating a new Document.
```

Comment:
Looks fine, but I'd suggest some small changes:

~~~

1. Connect to your appliance via IP or FQDN, for example, 172.16.40.100 or my_owncloud_appliance.
2. Login into the management console, for example, click on the domain and system settings, type in the Administrator as username and the Password you set.
3. Go to **Applications/Software** and then **Appcenter**
4. Search for Collabora Online Development Editon or select it from the applist.
5. Install the C.O.D.E

Now you can use Collabora within ownCloud.
Start by creating a new Document.

Reply:
Could you add some screen shots like in the rest of the appliance doc?